docgen/plugins/documentationjs-data.js

```diff
@@ -70,8 +72,14 @@ function filterSymbolsByType(type, symbols) {
   });
 }
 
+function groupSymbolsByCategories(symbols) {
+  return groupBy(symbols, (s) => {
+    const tag = filter(s.tags, {title: 'category'})[0];
+    return tag && tag.description || 'other';
```

Comment:
@Haroenv Not really, it can fail when the first tag found is undefined. You can fix it this way:

```js
const [ { description = 'other' } = {} ] = filter(s.tags, { title: 'category' });
return description;
```

And even this can fail if the first tag found is not an object or not undefined. I would not recommend to go this far in object destructuring if your are not sure of the result.